check copyrights, attribution and licencing

paolo-caroni opened the pull request:
as you requested I tried to make order in the license of the HURD table.
actually I realized that the GNU and HURD logo are not under GPL.

rzr committed:
hurd: branch: pinball-data-extra provide a hurd table

rzr committed:
hurd: build: maemo: freemantle (x86)

rzr committed:
hurd: imported hurd table from Paolo Caroni archive

see 'data/hurd/README.txt'

rzr committed:
build: Add SDL macros to acinclude.m4

Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
docker: Build in container

Useful for integration test

Usage:

    docker-compose up --build

Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
legal: Set GPL-2+ licence aligned to upstream

Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
init:

Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
git: Ignore autotools files

Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
hurd: build: Add autotools build files

Bug: https://github.com/rzr/pinball/issues/4
Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
debian: Initial packaging

This may be relocated to downstream/debian,
once shipped in debian

Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
build: Add helper tool to rebuild all

Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
build: Use pkg-config for sdl

This was needed by bitbake

Change-Id: I429a930ece6eb82b4070db9995455cae0b79efaa
Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
module: Use headers from namespaces

It will be easier to support various buildsystem

Change-Id: I86c01d3c2714ca2b71352718b9407e60222e60fd
Signed-off-by: Philippe Coval <rzr@users.sf.net>

rzr committed:
build: Prefer local sdl.m4

Signed-off-by: Philippe Coval <rzr@users.sf.net>

mend-bolt-for-github[bot] committed:
Add .whitesource configuration file

rzr committed:
doc: Add info about building

Signed-off-by: Philippe Coval <rzr@users.sf.net>

paolo-caroni committed:
Update README.txt for forced change of licence

GNU and HURD logo aren't released under GPLv2

paolo-caroni committed:
Update copyright

Update and correct copyright notice

paolo-caroni committed:
Merge pull request #1 from paolo-caroni/patch-1

Update README.txt for forced change of licence

paolo-caroni committed:
Merge pull request #2 from paolo-caroni/patch-2

Update copyright

rzr commented on debian/copyright:
```diff
@@ -2,12 +2,16 @@ Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-License: (GFDL or Free Art or CC-BY-SA) and GPLv3 and CC-BY-SA
- See data/hurd/README
- .
+Files: data/hurd/*
```
* will not work here because we have several licenses

rzr commented on debian/copyright:
```diff
@@ -2,12 +2,16 @@ Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Copyright: 2010 Paolo Caroni <kenren89@gmail.com>
+           2001 Henrik Enqvist <henqvist@excite.com>
+License: GPL-2+, CC-BY-SA-2.0, CC-BY-SA-3.0
```
License: should contain only one keyword

So you need to list them separately (without *)

rzr commented on debian/copyright:
```diff
@@ -2,12 +2,16 @@ Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+ the file ModuleHurd.cpp and tiles.png are derived from the tux table,
+ much of the sound are simply copied from tux table.
+ 
+ 
```
I dont think the above block is expected here

rzr reviewed (changes requested):
see example 4 in

https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#examples

paolo-caroni committed:
modified for conformance debian policy

I hope that I have correct the file,
i'm not sure if the licence CC-BY-SA needs to be copied or it's ok put only a link

paolo-caroni reviewed (commented):
modified in conformity of example 4 in

https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#examples

rzr commented on debian/copyright:
```diff
@@ -1,20 +1,65 @@
+License: GPL-2+
+
+License: CC-BY-SA-2.0
+https://spdx.org/licenses/CC-BY-SA-2.0
```
Unsure link is enough maybe you need to copy bit, 
let's see what lintian says

rzr commented on debian/copyright:
```diff
@@ -1,20 +1,65 @@
+       data/hurd/sub.png
+       data/hurd/tiles2.png
+       data/hurd/Wood.jpg
+       data/hurd/freesoftwaresong.xml
```
maybe RMS should be mentioned as author of song ?

paolo-caroni replied:
ok, since the original song is public domain I've simply skipped it, but you are right.
I have write it on musescore however RMS needs to be mentioned.

rzr reviewed (commented):
Looks far better, maybe I should setup CI to see if lintian is reporting any issue

Meanwhile @zezinho42 or anyone can review.

paolo-caroni committed:
added RMS

paolo-caroni reviewed (commented):
add RMS

rzr commented:
sorry I didnt want to close with a push force, are you able to reopen ?
